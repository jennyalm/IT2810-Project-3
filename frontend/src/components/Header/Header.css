.header {
    background-color: #1c272b;
    min-height: 10vh;
    display: flex;
    flex-direction: column;
    align-content: center;
    padding-left: 2vw;
    color: white;
    letter-spacing: 1vh;
  }

.App-header {
    align-self: center;
}

.toggle {
    align-self: center;
    width: 100%;
}