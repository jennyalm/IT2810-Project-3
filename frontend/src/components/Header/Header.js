import React from "react";
import './Header.css';
import StatToggle from "../StatButton/StatButton";

const Header = (props) => {
    return (
        <div className="header">
            <header className="App-header">
                <h1>{props.text}</h1>
            </header>
            <div className="toggle">
                <StatToggle/>
            </div>
        </div>

    );
};

export default Header;