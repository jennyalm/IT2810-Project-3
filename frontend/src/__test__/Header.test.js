import React from 'react'
import renderer from 'react-test-renderer'


import Header from "../components/Header/Header"

test('Header snapshot test', () => {

    const tree = renderer.create(<Header />)

    expect(tree).toMatchSnapshot()

})
