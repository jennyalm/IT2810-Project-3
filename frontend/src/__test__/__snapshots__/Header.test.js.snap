// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot test 1`] = `
<header
  className="App-header"
>
  <h2 />
</header>
`;
